Add README around setting up zipkin (#5545)
solo-io/gloo: docs/content/guides/observability/tracing.md

        prefixRewrite: /
status: # collapsed for brevity
{{< /highlight >}}

##### 4. (Optional) Setting up zipkin locally

> 

Reviewer: Could use some context around why we might want to do this, and to clarify that this assumes we have a kind cluster with gloo running, even if it seems obvious
Something like:
"These instructions describe how to set up a simple tracing solution to experiment with if developing locally in a kind cluster (see <link to hello world guide>)"

Reviewer 2: ```suggestion
Set up Zipkin tracing in a [local Kind cluster]({{< versioned_link_path fromRoot="/installation/platform_configuration/cluster_setup/#kind" >}}) for local troubleshooting and experimentation. 
```

> `docker run --network=kind -itd --name zipkin -p 9411:9411 openzipkin/zipkin`

Reviewer 2: ```suggestion
1. Run Zipkin.
    ```shell
    docker run --network=kind -itd --name zipkin -p 9411:9411 openzipkin/zipkin
    ```
```

> Run `docker network inspect kind` to ensure both zipkin and zipkin-tracing-control-plane are in the kind network.

Reviewer 2: ```suggestion
2. Verify that both `zipkin` and `zipkin-tracing-control-plane` are in your local Kind cluster network.
     ```shell
     docker network inspect kind
     ```
```

> ```json

Reviewer 2: ```suggestion
{{< highlight json "hl_lines=35">}}
```

        },
        "Internal": false,
        "Attachable": false,
        "Ingress": false,
        "ConfigFrom": {
            "Network": ""
        },
        "ConfigOnly": false,
        "Containers": {
            "3431770d0c41bfbc8eceac4c806605286f5dac81820599f828dcb250037a2f48": {
                "Name": "zipkin-tracing-control-plane",
                "EndpointID": "3e48e18bc7b259ca9d597a594ee3d5205c8339e8ecd9f8f274a178d07f395b78",
                "MacAddress": "02:42:ac:12:00:03",
                "IPv4Address": "172.18.0.3/16",
                "IPv6Address": "fc00:f853:ccd:e793::3/64"
            },
            "84dadbd86f113c7104eca23d3d78e9dec997a47666c1ba4eed2ae7a5ad8eb20d": {
                "Name": "zipkin",
                "EndpointID": "09e07c8ac6b1cd912c325962586d9497520e216a4ec357384c663594248fc104",
                "MacAddress": "02:42:ac:12:00:02",
                "IPv4Address": "172.18.0.2/16",
                "IPv6Address": "fc00:f853:ccd:e793::2/64"
            }
        },
        "Options": {
            "com.docker.network.bridge.enable_ip_masquerade": "true",
            "com.docker.network.driver.mtu": "1500"
        },
        "Labels": {}
    }
]

> ```

Reviewer 2: ```suggestion
{{< /highlight >}}
```

> `172.18.0.2` would be the IP to specify as the address for your zipkin cluster [here](https://docs.solo.io/gloo-edge/latest/guides/observability/tracing/#1-configure-a-tracing-cluster)

Reviewer 2: ```suggestion
3. Configure a [tracing cluster]({{< versioned_link_path fromRoot="/guides/observability/tracing/#1-configure-a-tracing-cluster" >}}) with the IP address of Zipkin, such as `172.18.0.2` in this example.
```
